Automattic/jetpack
# VideoPress: show Settings only when user is connected

This PR shows Settings only when the user is connected. Also, it cleans duplicated code.

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: show Settings only when user is connected

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to VideoPress dashboard 
* When connected, confirm you see the Settings section
* When not connected, confirm you don't see the Settings section
